contracts/Bond.sol

```diff
@@ -73,6 +73,10 @@ contract Bond is
         _;
     }
 
+    constructor() {
```

Comment:
```suggestion
    constructor() {
     /*
        Since the constructor is executed only when creating the
        implementation contract, prevent its re-initialization.
        
    */
```